build/charts/antrea/crds/bgppolicy.yaml

```diff
@@ -0,0 +1,128 @@
+                  properties:
+                    matchExpressions:
+                      items:
+                        properties:
+                          key:
+                            type: string
+                          operator:
+                            enum:
+                              - In
+                              - NotIn
+                              - Exists
+                              - DoesNotExist
+                            type: string
+                          values:
+                            items:
+                              type: string
+                              pattern: "^(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?$"
+                            type: array
+                        type: object
+                      type: array
+                    matchLabels:
+                      x-kubernetes-preserve-unknown-fields: true
+                localASN:
+                  type: integer
+                  minimum: 64512
+                  maximum: 65535
+                  format: int32
+                  default: 64512
+                listenPort:
+                  type: integer
+                  minimum: 1
+                  maximum: 65535
+                  format: int32
+                  default: 179
+                advertisements:
+                  type: object
+                  properties:
+                    service:
+                      type: object
+                      anyOf:
+                        - required: [ clusterIPs ]
+                        - required: [ externalIPs ]
+                        - required: [ loadBalancerIPs ]
+                      properties:
+                        clusterIPs:
+                          type: boolean
+                        externalIPs:
+                          type: boolean
+                        loadBalancerIPs:
+                          type: boolean
+                    pod:
+                      type: object
+                      properties: {}
+                    egress:
+                      type: object
+                      properties: {}
+                bgpPeers:
+                  type: array
+                  items:
+                    type: object
+                    required:
+                      - address
+                      - asn
+                    properties:
+                      address:
+                        type: string
+                        format: cidr
+                      port:
+                        type: integer
+                        default: 179
+                        format: int32
+                        minimum: 1
+                        maximum: 65535
+                      asn:
+                        type: integer
+                        minimum: 1
+                        maximum: 65535
+                        format: int32
+                      multihopTTL:
+                        type: integer
+                        default: 1
```

Comment:
One nit, you defined the default value in different orders for different fields, better to keep them the same formats as K8s or othter Antrea CRDs.

Reply:
Will correct.